server/controllers/clap.controller.js

```diff
@@ -0,0 +1,38 @@
+import db from '../models';
+import Response from '../utils/response.util';
+
+const { Clap, Article } = db;
+/**
+ * Clap Controller
+ * @package clap
+ */
+class ClapController {
+  /**
+   * @description - Controller for create a clap endpoint
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   * @memberof createClap
+  */
+  static async createClap(req, res) {
+    const {
+      articleId
+    } = req.body;
+    const { id } = req.user;
+    const successMessage = 'Clap successfully created';
```

Comment:
You are using this variable once. No need to have it here. Use it directly where it should be used